### ISSUE TITLE
TypeScript type error possibly caused by type from node perf_hooks that is not imported

### ISSUE THREAD

**Issue by sjsyrek** [getsentry/sentry#17650]
## Important Details

How are you running Sentry?

* [ ] On-Premise docker [Version xyz]
* [ ] Saas (sentry.io)
* [x] Not running it at all, because I can't even compile my project

## Description

When I try to compile my TypeScript project, which uses Sentry, I get this type error:
```
../../node_modules/@sentry/utils/dist/misc.d.ts:90:53 - error TS2304: Cannot find name 'Performance'.

90 export declare const crossPlatformPerformance: Pick<Performance, 'now' | 'timeOrigin'>;
                                                       ~~~~~~~~~~~
```

I checked the file referenced above, and my guess is that `Performance` type needs to be imported from node somehow. It appears to exist in the npm `@types/node` package under `perf_hooks.d.ts`.

## Steps to Reproduce

I am not sure how to reproduce this or whether it's a problem with my project or Sentry itself.

### What you expected to happen

Using Sentry in TypeScript files should not cause type errors.

### Possible Solution

Possibly import the above type into the file causing the problem.

**Comment by doudejans** [comment]
Hi @sjsyrek,
I just started looking into this issue over at the [sentry-javascript](https://github.com/getsentry/sentry-javascript) repository when I noticed that it has already been fixed in [this PR](https://github.com/getsentry/sentry-javascript/pull/2495)! The fix seems to be included in [version 5.14.2](https://github.com/getsentry/sentry-javascript/releases/tag/5.14.2) of the SDKs, so please try to update.

**Comment by sjsyrek** [comment]
Thank you so much! 😎
